### PR TITLE
Add chain config for Sei pacific-1

### PR DESCRIPTION
**Pull request by besated**

### CODE REVIEW

File: cosmos/pacific.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
+  "rpc": "https://rpc.wallet.pacific-1.sei.io",
```
Comment (codebycarson) [inline cosmos/pacific.json:2]: Let's add a public provider like Rhino here? We don't want people using this url except wallet providers.

File: cosmos/pacific.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  "bech32Config": {
+    "bech32PrefixAccAddr": "sei",
+    "bech32PrefixAccPub": "seipub",
+    "bech32PrefixValAddr": "seivaloper",
+    "bech32PrefixValPub": "seivaloperpub",
+    "bech32PrefixConsAddr": "seivalcons",
+    "bech32PrefixConsPub": "seivalconspub"
+  },
+  "nodeProvider": {
+        "name": "Sei",
```
Comment (codebycarson) [inline cosmos/pacific.json:24]: Let's put Rhino here

File: cosmos/pacific.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      "coinMinimalDenom": "factory/sei189adguawugk3e55zn63z8r9ll29xrjwca636ra7v7gxuzn98sxyqwzt47l/9tTHn18vLnfyBvrQaia6N15zwrfRCAebZDshoPZ39ahN",
+      "coinDecimals": 8,
+      "coinGeckoId": "weth"
+    },
+        {
+      "coinDenom": "USDTbs",
+      "coinMinimalDenom": "factory/sei189adguawugk3e55zn63z8r9ll29xrjwca636ra7v7gxuzn98sxyqwzt47l/871jbn9unTavWsAe83f2Ma9GJWSv6BKsyWYLiQ6z3Pva",
+      "coinDecimals": 8,
+      "coinGeckoId": "tether"
+    }
```
Comment (HeesungB) [inline cosmos/pacific.json:105]: @besated Is this factory tokens received from bridge?
or Is is new token?

Comment (besated) [reply cosmos/pacific.json:105]: yes, these are factory tokens that are from bridging

Comment (HeesungB) [reply cosmos/pacific.json:105]: @besated What it different between WETH and WETHbs.
What is the difference between USDCet, USDCar, USDCpo, USDCop, USDCso, and USDCbs?

**Comment by HeesungB** [comment]
@besated Thank you for contribution!

**Pull request merged**